tests/unit/datasets/datasets_test.py

```diff
@@ -31,6 +31,7 @@
     ('public_dataset', 'dataset_name'),
     # XXX: add public dataset in alphabetical order
```

Comment:
this comment here should probably removed, right? It was introduced by https://github.com/aeye-lab/pymovements/commit/054d567fe5c5e9b8f42c2ee193eb5a8b61f3c993

Reply:
I think we can leave it but maybe remove the `XXX`?

Reply:
# please add datasets in alphabetical order